cmd/genji/dump.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/genjidb/genji/engine/badgerengine"
+	"github.com/genjidb/genji/engine/boltengine"
+
+	"github.com/dgraph-io/badger/v2"
+	"github.com/genjidb/genji/engine"
+
+	"github.com/genjidb/genji"
+	"github.com/genjidb/genji/cmd/genji/shell"
+)
+
+func executeDump(ctx context.Context, f string, tables []string, e string, dbPath string, w io.Writer) error {
+	if dbPath == "" {
+		return errors.New("db path should be specified")
+	}
```

Comment:
I think that this can be checked earlier in the execution flow.

Reply:
At the first I did the check in the main, but after some reflection I thought it is better to check in the here. Let's imagine tomorrow we write a function or command that call `executeDump`?

Reply:
> should we trust the caller

Since that we have only one entry point and we are the caller I think we can trust it.

> Do you think it 's better to make the check before?

Honestly, I don't have a strong opinion on this, it's up to you.